versioned_docs/version-1.5/release-notes.md

```diff
@@ -17,10 +17,10 @@ Here's the different components, their latest version and a link to the respecti
 
 | Name                                                                 | Version | Release Notes                                                              |
 |----------------------------------------------------------------------|---------|----------------------------------------------------------------------------|
-| [Elemental Operator](https://github.com/rancher/elemental-operator/) | v1.5.4  | [Link](https://github.com/rancher/elemental-operator/releases/tag/v1.5.4)  |
-| [Elemental Toolkit](https://github.com/rancher/elemental-toolkit/)   | v1.1.5  | [Link](https://github.com/rancher/elemental-toolkit/releases/tag/v1.1.5)   |
+| [Elemental Operator](https://github.com/rancher/elemental-operator/) | v1.6.3  | [Link](https://github.com/rancher/elemental-operator/releases/tag/v1.6.3)  |
```

Comment:
These release notes for v1.5 should remain unchanged.

Reply:
Good catch, I meant to update the "next" ones, not these. :D